app/default/src/Elasticsearch/Repository/CarRepository.php

```diff
@@ -13,17 +13,20 @@
 use App\ValueObject\Cars;
 use App\ValueObject\CriteriaInterface;
 use App\ValueObject\Pagination;
+use Psr\Log\LoggerInterface;
 
 final class CarRepository implements CarRepositoryInterface
 {
 	private Index $index;
 
 	public function __construct(
 		private ApiClientInterface    $client,
-		private CarsHydratorInterface $hydrator
+		private CarsHydratorInterface $hydrator,
+		LoggerInterface $logger = null
```

Comment:
Missing visibility operator

Reply:
Thank you for spotting it. Done in [59e47a7](https://github.com/lrynek/painless-car-rental/pull/1/commits/59e47a75572e500cd7070e0f4740bb8ed76843cd)